### DISCUSSION TITLE
RootResults.__init__ error

### DISCUSSION THREAD

**Discussion by Elisa-Visentin** [gammapy/gammapy#5034] [Help] [answered]
Hi all,
Sorry for disturbing you... I have some issues with gammapy 1.1. I get this error while running this line of code:
```
../../../micromamba/envs/magic-lst/lib/python3.10/site-packages/gammapy/utils/roots.py:9: in <module>
    BAD_RES = RootResults(root=np.nan, iterations=0, function_calls=0, flag=0)
E   TypeError: RootResults.__init__() missing 1 required positional argument: 'method'
```
Seems like an issue with the scipy dependency version. Do you know which is the last scipy version compatible with this version of Gammapy? And, really sorry for that, but... could you fix this?

Thanks

**Comment by steob92** [comment, marked as answer]
I had the same issue. Looks like Scipy 1.12.0 was released a [few days ago](https://scipy.org/news/#scipy-1120-released). For the moment downgrading to scipy 1.11.4 removes the issue. Something like:
```
mamba install "scipy==1.11.4"
```

**Reply by Elisa-Visentin** [reply]
Thank you
